frontend/server/k8s-helper.ts

```diff
@@ -146,10 +147,9 @@ export async function getTensorboardInstance(
           viewer.body.spec.type === 'tensorboard'
         ) {
           const address = `http://${viewer.body.metadata.name}-service.${namespace}.svc.cluster.local:80/tensorboard/${viewer.body.metadata.name}/`;
-          const version = viewer.body.spec.tensorboardSpec.tensorflowImage
-            ? viewer.body.spec.tensorboardSpec.tensorflowImage.replace('tensorflow/tensorflow:', '')
-            : '';
-          return { podAddress: address, tfVersion: version };
+          const tfImageParts = viewer.body.spec.tensorboardSpec.tensorflowImage.split(':', 1);
```

Comment:
This line has a bug:
```
'tensorflow:2.0.0'.split(':', 1) === ["tensorflow"]
```
So the next line of tfVersion will always be empty, because tfImageParts.length is always 1.

Reply:
You're right. This used to not have the `1`. I misunderstood the documentation. Not sure how this ever worked. Sending a patch...

Reply:
FYI https://github.com/kubeflow/pipelines/pull/3356